.gitlab/stream-ci.yaml

```diff
@@ -0,0 +1,259 @@
+stages:
```

Comment:
I wanted to keep it, in case someone wants to test on GitLab infrastructure on their own machines. However, these scripts aren't run in production, hence inevitably will slip out of sync which will reduce it's value over time. I guess it's up to the group whether they see value in keeping them inside the repo. As far as Stream is concerned, we're fine either way.

Reply:
I went ahead and removed. We can make it the topic of a separate discussion whether we want to host such scripts. We still have them, so it won't evaporate.